packages/common/package.json

```diff
@@ -7,6 +7,9 @@
     ".": "./index.ts",
     "./refresh-runtime": "./refresh-runtime.js"
   },
+  "dependencies": {
+    "@rolldown/pluginutils": "1.0.0-beta.41"
```

Comment:
This should be added to SWC deps too (and maybe rsc) because the common package in bundled into react plugins. I'm not a power user of pnpm, but is catalog the best way to unsure it's the same version everywhere?

Also should we use a strict version?

Reply:
Yes ok with dependabot this should be done in sync 👍